recipe/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyxirr" %}
+{% set version = "0.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pyxirr-{{ version }}.tar.gz
+  sha256: 8065865c327afef271dd5cd8561af22f407ccfde352b15860fe5aa0daee0af5d
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True #  [py37 or py>312]
+  missing_dso_whitelist:
+    - $RPATH/ld64.so.1  # [s390x]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+  host:
+    - python
+    - maturin >=1,<2
+    - pip
+    - setuptools
+    - wheel
```

Comment:
```suggestion
```
maturin is the build backend and is sufficient.

Reply:
Thank you! First time building a rust package. Is this normal when it comes to rust packages?

Reply:
Any package using another build backend such as `maturin` or `hatchling` wouldn't need `wheel`